### ISSUE TITLE
patterns are not working properly

### ISSUE THREAD

**Issue by renciebautista** [BBFMedia/PHPJasperXML#9]
patterns made on the ireport app is not working when pdf file is generated

**Comment by hidayat88** [comment]
Yes, true, I also getting this issue, to implement date format. Currently only support standard format like dd/MM/yy, h:m:s a, but cannot customize more complex format
